.git-hooks/pre-commit

```diff
@@ -129,6 +129,17 @@ sub check_author()
     }
 }
 
+sub check_prettier()
+{
+    my $prettier = `make prettier`;
+
+    if ($prettier != 0)
+    {
+        print("Your code is not formatted. Please run 'make prettier-write' before committing.");
+        exit(1);
```

Comment:
Currently, the print command only shows as a warning or normal message. It might be more effective to use the `die` command instead.

For example: ```die "ERROR: Your code is not formatted. Please run 'make prettier-write' before committing.\n";```